src/main/java/com/currencycloud/client/CurrencyCloud.java

```diff
@@ -54,17 +54,22 @@ Account createAccount(
             @HeaderParam("X-Auth-Token") String authToken,
             @HeaderParam("User-Agent") String userAgent,
             @FormParam("account_name") String accountName,
-            @Nullable @FormParam("legal_entity_type") String legalEntityType,
+            @FormParam("legal_entity_type") String legalEntityType,
+            @FormParam("street") String street,
+            @FormParam("city") String city,
+            @FormParam("postal_code") String postalCode,
+            @FormParam("country") String country,
+            @Nullable @FormParam("state_or_province") String stateOrProvince,
+            @Nullable @FormParam("brand") String brand,
             @Nullable @FormParam("your_reference") String yourReference,
             @Nullable @FormParam("status") String status,
-            @Nullable @FormParam("street") String street,
-            @Nullable @FormParam("city") String city,
-            @Nullable @FormParam("state_or_province") String stateOrProvince,
-            @Nullable @FormParam("postal_code") String postalCode,
-            @Nullable @FormParam("country") String country,
             @Nullable @FormParam("spread_table") String spreadTable,
             @Nullable @FormParam("identification_type") String identificationType,
-            @Nullable @FormParam("identification_value") String identificationValue
+            @Nullable @FormParam("identification_value") String identificationValue,
+            @Nullable @FormParam("api_trading") Boolean apiTrading,
+            @Nullable @FormParam("online_trading") Boolean onlineTrading,
+            @Nullable @FormParam("phone_trading") Boolean phoneTrading
+
```

Comment:
Changing the order of parameters to a core call is a breaking change. Major version dump and documentation rewrite (with special large highlighted box for this change)?

Reply:
Order changed for URL parameters only. Signature of core call "public Account createAccount(Account account)" in CurrencyCloud.java continues to be same